src/runner.ts

```diff
@@ -1,3 +1,112 @@
+import * as dopri from "dopri";
```

Comment:
So dopri is going to get bundled with the runner and won't need to be passed as a paramter by the front end any more?

Reply:
yeah, that's right - unless you prefer some other way?

Reply:
Nope, sounds good to me. I guess passing it in the front end keeps down the payload size, but this way means this package can control which version to use.

Reply:
This should all be something that we pass to the client just once per session I think, and we're looking at about 13KB so far (this code + the dopri lib) so it's only 3x larger than the sid logo :)